core/lib/dal/src/blocks_dal.rs

```diff
@@ -1090,12 +1090,25 @@ impl BlocksDal<'_, '_> {
     }
 
     /// This method returns batches that are confirmed on L1. That is, it doesn't wait for the proofs to be generated.
+    ///
+    /// # Params:
+    /// * `commited_tx_confirmed`: whether to look for ready proofs only for txs for which
+    ///   respective commit transactions have been confirmed by the network.
     pub async fn get_ready_for_dummy_proof_l1_batches(
         &mut self,
+        commited_tx_confirmed: bool,
```

Comment:
nit: I think something like `only_commited_batches` would be more self-explanotary